cvise/utils/testing.py

```diff
@@ -329,8 +318,8 @@ def diff_files(orig_file, changed_file):
     def check_sanity(self, verbose=False):
         logging.debug('perform sanity check... ')
 
-        folder = tempfile.mkdtemp(prefix=f'{self.TEMP_PREFIX}sanity-')
-        test_env = TestEnvironment(None, 0, self.test_script, folder, None, self.test_cases, None)
+        folder = Path(tempfile.mkdtemp(prefix=f'{self.TEMP_PREFIX}sanity-'))
+        test_env = TestEnvironment(None, 0, self.test_script, folder, list(self.test_cases)[0], self.test_cases, None)
```

Comment:
I think this repeats the first test case.  Not sure if it matters, I'm just pointing it out here in case I run into it when I get round to testing.

Reply:
Yes, it's intentional.